Hudl.Mjolnir/Command/Attribute/FireAndForgetAttribute.cs

```diff
@@ -13,7 +13,7 @@ namespace Hudl.Mjolnir.Command.Attribute
     /// 
     /// Should only be used on interface methods.
     /// </summary>
-    [Obsolete("Use [FireAndForget] from the Hudl.Mjolnir.Attributes package instead")]
+    [Obsolete("Will be removed in a future release. Leaving async calls un-awaited is generally not recommended.")]
```

Comment:
Or I suppose you want to do that in a major release (and only make this a minor release)?